### ISSUE TITLE
Models for Self-Supervised Downstream Detection Tasks

### ISSUE THREAD

**Issue by chos1npc** [open-mmlab/mmdetection#10831]
I would like to ask whether it is possible to train the model transferred from self-supervision through mmdet. Is the naming of the backbone consistent with most self-supervised models?

**Comment by hhaAndroid** [comment]
@chos1npc Self-supervised training is currently done using mmpretrain, and the resulting models can be directly used in MMDetection. You can perform self-supervised training using the mmpretrain library and then directly utilize the trained models in the MMDetection framework for tasks such as object detection.

https://github.com/open-mmlab/mmpretrain/blob/main/docs/en/user_guides/downstream.md

**Comment by chos1npc** [comment]
@hhaAndroid Is it possible to transfer a self-supervised model that wasn't trained using mmpretrain?

**Comment by hhaAndroid** [comment]
@chos1npc Of course, you can simply transfer some keys or directly copy the backbone code and register it to achieve that.
